main.js

```diff
@@ -93,6 +98,6 @@ app.on('ready', function(){
     app.quit();
   });
 
-  app.dock.hide();
+  //app.dock.hide();
```

Comment:
In order to avoid OSX issues this can be:

    if (app.dock) { app.dock.hide(); }